src/Path/Include.hs

```diff
@@ -482,9 +482,11 @@ splitExtension (Path fpath) =
             trailingSeps = takeWhile isSep rstr
             xtn  = (takeWhile notSep . dropWhile isSep) rstr
         in (reverse name, reverse xtn ++ trailingSeps)
-    normalizeDrive
-        | IS_WINDOWS = normalizeTrailingSeps
-        | otherwise  = id
+#if IS_WINDOWS
```

Comment:
What was this change for?

Reply:
Ah, ok, that's why it was solved this way! I will change that to use the newly added `OsPath.Internal.{Posix,Windows}.isWindows`.

Reply:
BTW: Which code formatter is used by this project? I assume since Chris Done started it it's `hindent`?
Also, should I format the Cabal file with something like `cabal-fmt` or `cabal-gild` or not?

Reply:
@mmhat Right, it gets compiled away anyway.
Right now there is no formatter for this project so don't worry about formatting it right now.

Reply:
I removed that CPP in the recent commits.